lib/matplotlib/patches.py

```diff
@@ -181,14 +181,29 @@ def contains(self, mouseevent, radius=None):
 
     def contains_point(self, point, radius=None):
         """
-        Returns *True* if the given point is inside the path
+        Returns `True` if the given point is inside the path
```

Comment:
Sorry to be a pain:
- Bits of code (like True) should have double ticks around them (``True``)
- Keyword arguments (like points below) should have stars (*points*)